### DISCUSSION TITLE
binomial issue

### DISCUSSION THREAD

**Discussion by NcyFS** [lindeloev/mcp#141] [General]
![image](https://user-images.githubusercontent.com/80974983/149555223-53cf7645-d3cf-409f-98f9-eac369c3e472.png)
![mcp1](https://user-images.githubusercontent.com/80974983/149555784-24683ec4-596b-45f7-9d99-e63b6fd9147e.PNG)
![mcp 2](https://user-images.githubusercontent.com/80974983/149555796-40f97b47-63e7-4cd9-b239-f2878cd6ce1c.PNG)
![mcp3](https://user-images.githubusercontent.com/80974983/149555800-47d41ef1-99ab-44e2-92e1-3e9eb776d0bf.PNG)

First of all, thank you so much for this wonderful package! I just have a question about the use of the binomial law. I have this error message, and I don't know what the source is.

**Comment by lindeloev** [comment]
Can you post your model? It seems that you have `survie` as the dependent variable. `survie` has decimals but the binomial family only allows for integers >= 0, c.f. the error message.

**Comment by NcyFS** [comment]
Thank you so much for all your help !!! indeed, I was confused.
